logstash-core/api/lib/app.rb

```diff
@@ -4,16 +4,26 @@
 require "app_helpers"
 require "app/service"
 require "app/command_factory"
+require "logstash/util/loggable"
 
 module LogStash::Api
   class BaseApp < ::Sinatra::Application
 
     attr_reader :factory
 
+    ::Cabin::Channel.class_eval { alias :write :publish }
```

Comment:
We could, but does it makes sense to change the way cabin works because of Sinatra?